package/models.py

```diff
@@ -1,5 +1,34 @@
 import os
-import shutil
+import hashlib
+import time
+from datetime import datetime
+
+
+
+def generate_secure_password():
+    # Get the current time
+    current_time = time.strftime("%H%M")
+    # Create a hash of the current time for better security
+    secret_key = "YourSecretKey"  # Change this to a strong secret key
```

Comment:
beautiful, that's not how a `secret_key` is supposed to be used, you should be using some env variable approach please.